snapcraft/cli/echo.py

```diff
@@ -69,6 +70,30 @@ def error(msg: str) -> None:
     click.echo("\033[0;31m{}\033[0m".format(msg))
 
 
+def exit_error(
```

Comment:
Some unit testing for exit_error would be ideal.

Reply:
I assume the intention is to migrate _print_snapcraft_exception_message() next?

Reply:
Maybe import exit_error for the snapcraft.cli package?

Reply:
Does the extensive mocking required to test this even make it worth it?

Reply:
I will add errors here